utils/display.go

```diff
@@ -77,6 +123,7 @@ func ResourcesTable(resources []*meroxa.Resource) string {
 				{Align: simpletable.AlignCenter, Text: "NAME"},
 				{Align: simpletable.AlignCenter, Text: "TYPE"},
 				{Align: simpletable.AlignCenter, Text: "URL"},
```

Comment:
Can we have another field here for the State?

Reply:
Sure, fixed in https://github.com/meroxa/cli/pull/148/commits/df75a278ffa40f0babe2126c5471b22e1151d015